### ISSUE TITLE
new install

### ISSUE THREAD

**Issue by mikeh86** [BenHUET/eveseat-calendar#7]
Using version ^1.3 for kassie/calendar
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 30 installs, 0 updates, 0 removals
  - Installing golonka/bbcodeparser (v2.2.2): Downloading (failed)    Failed to download golonka/bbcodeparser from dist: The "https://api.github.com/repos/golonka/BBCodeParser/zipball/769c4ebe6207ffa20298b84b90eafca87ce2fb95" file could not be downloaded (HTTP/1.1 404 Not Found)
    Now trying to download from source
  - Installing golonka/bbcodeparser (v2.2.2): Cloning 769c4ebe62

  [RuntimeException]
  Failed to clone https://github.com/golonka/BBCodeParser.git via https, ssh protocols, aborting.
  - https://github.com/golonka/BBCodeParser.git
    remote: Invalid username or password.
    fatal: Authentication failed for 'https://github.com/golonka/BBCodeParser.git/'
  - git@github.com:golonka/BBCodeParser.git
    Permission denied (publickey).
    fatal: Could not read from remote repository.

    Please make sure you have the correct access rights
    and the repository exists.

**Comment by BenHUET** [comment]
yup it seems the guy deleted his github repo so composer point to nothing... I'll check this soon 

thanks for the feedback

**Comment by mikeh86** [comment]
@BenHUET thank you for getting back to me so quickly, I see other devs are doing BBCodeParser could it be worth removing in install of it from your installer and get people to install another version?

**Comment by warlof** [comment]
@mikeh86 code need to be updated in some place in order to handle new parser library.
@BenHUET this one seems to be maintained and widely used : https://github.com/s9e/TextFormatter

**Comment by BenHUET** [comment]
should be resolved now in version 1.3.1
